packages/components/src/Datalist/Datalist.component.js

```diff
@@ -170,6 +170,10 @@ class Datalist extends Component {
 		if (this.props.multiSection) {
 			newValue = this.state.suggestions[sectionIndex].suggestions[itemIndex];
 		}
+		if (newValue.disabled) {
```

Comment:
We need to follow that PR I guess https://github.com/moroshko/react-autowhatever/pull/35

Reply:
Yes, also saw that PR, but it was done around 4 months ago (
But we can follow of course

Reply:
Yes, also saw that PR, but it was done around 4 months ago (
But we can follow of course